### DISCUSSION TITLE
2.3.4.RELEASE

### DISCUSSION THREAD

**Discussion by Aloren** [aerospike/spring-data-aerospike#529] [Team Posts]
Hi @rbotzer !
Could you please ask the guys to trigger a release for us. Thanks!

PS: I've migrated the build to gh actions, so in case you want to publish releases to gh packages it will be easy to do :)

**Comment by rbotzer** [comment]
Brian published the 2.3.4 build. What do you need me to do to  get builds onto the GitHub Packages for this repo?  Or is that something you can do?

Our goal is to move away from Sonatype and use GitHub. I don't think there's any strong need for one over the other. If there is, can you clarify why?

Basically, I want you to be empowered to publish binaries for the releases to https://maven.pkg.github.com/aerospike-community . So let's try that, and if you run into any permissions issues, let's solve them when they happen. 
Please let me know if it works for you. Thanks!

**Comment by Aloren** [comment]
I don't know yet, I will need to check :)
